src/components/signal-viewer/index.css

```diff
@@ -1,42 +1,6 @@
 .signal-viewer {
-  margin: 10px;
-  width: fit-content;
-  max-width: 100%;
-  font-size: 0.9em;
-  overflow: auto;
-  flex: 1;
-}
-
-.signal-viewer table {
-  max-width: 100%;
-  table-layout: auto;
-  border-collapse: collapse;
-}
-
-.signal-viewer th tr:first-child > * {
-  border-top: 0;
-  border-bottom: 2px solid #dfe2e5;
-}
-
-.signal-viewer th {
-  text-align: center;
-  background: #f6f8fa;
-}
-
-.signal-viewer td,
-.signal-viewer th {
-  min-width: 30px;
-  color: #333;
-  padding: 8px;
-  word-break: break-all;
-  border: 1px solid #dfe2e5;
-}
-
-.signal-viewer th,
-.signal-viewer td:first-child {
-  word-break: keep-all;
-}
-
-.signal-viewer tr:nth-child(odd) {
-  background-color: #f6f8fa;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding: 10px 20px 15px 20px;
```

Comment:
I think it was better before. Why did you change this?

Reply:
This was because when we scrolled to bottom, we couldn't see the last entry in the table. This ensured that the even some part of table will never go out of the view.

Reply:
I have not had that problem since my fixes in master.

Reply:
### master
![Screenshot from 2019-03-25 10-22-22](https://user-images.githubusercontent.com/35191225/54895933-e81ae780-4ee7-11e9-93ac-a14fb60dc171.png)

### scroll-bar-style
![Screenshot from 2019-03-25 10-20-58](https://user-images.githubusercontent.com/35191225/54895954-f9fc8a80-4ee7-11e9-8a45-e869aa947a28.png)

Notice the right padding.

Reply:
This is not a good enough fix for this issue. See how the tabs are over the table. Please revert your change and fix the issue in a new PR.

Reply:
I'll combine these three tabs in a hamburger and submit a new PR?
These changes only make the vertical scroll bar visible so that user can know it's a scrollable element.

Reply:
@domoritz Reverting it will cause some problems. Before this commit was made, the structure of the signal-viewer was like this:-
```
 <div className="signal-viewer">
        <table>
          <tbody>
            <tr>
              <th>Signal</th>
              <th>Value</th>
            </tr>
            {this.getSignals().map(signal => (
              <SignalRow key={signal} signal={signal} view={this.props.view} />
            ))}
          </tbody>
        </table>
      </div>
```
But now it has been changed to
```
 <div className="signal-viewer">
        <div className="editor-table">
          <table>
            <tbody>
              <tr>
                <th>Signal</th>
                <th>Value</th>
              </tr>
              {this.getSignals().map(signal => (
                <SignalRow key={signal} signal={signal} view={this.props.view} />
              ))}
            </tbody>
          </table>
        </div>
      </div>
```

Reply:
Why not just make this

```
<div className="signal-viewer">
          <table className="editor-table">
            <tbody>
              <tr>
                <th>Signal</th>
                <th>Value</th>
              </tr>
              {this.getSignals().map(signal => (
                <SignalRow key={signal} signal={signal} view={this.props.view} />
              ))}
            </tbody>
          </table>
      </div>
```

Reply:
Good idea! I'll try that remove the unnecessary styles.

Reply:
@domoritz Seems like the extra div acted as a container from which if the table overflowed, scroll bars would appear.

Reply:
I see. I think the table style itself should not have that role. See how I handled it for the signal table.

Reply:
The extra div was the place the scrollbars were attached. If I apply `overflow: auto` to the outermost div, since it takes the whole width of the container, the scrollbars will then appear sticking to the the container. The extra div is just a place for holding the scroll bars.

Reply:
That's not true as you can see on master right now. You can handle overflow differently for the signal table and the data viewer.

Reply:
I copied the same styles and structure for signal and data viewer that's why they both were working.
Plus data viewer also had the the extra div.

Reply:
The data viewer needed it because it has the additional buttons. The signal viewer does not.

Reply:
```
<div className="data-table">
    <table>
      <thead>
        <tr>{headerNodes}</tr>
      </thead>
      <tbody>{tableBody}</tbody>
    </table>
</div>
```
I was talking about this div.